app.js

```diff
@@ -1,17 +1,29 @@
 const express = require('express');
 const app = express();
 const bodyParser = require('body-parser');
-const routes = require('./routes/routes');
+const routes = require("./routes/neonRoute");
+
+//path to neon route
+const neonRoute = require('./routes/neonRoute');
+
```

Comment:
You can include the route in the routes.js file itself. Another file for this route specifically is not required.